index.html

```diff
@@ -35,6 +35,7 @@
       caniuse: "screen-orientation",
     };
     </script>
+    <link rel="stylesheet" type="text/css" href="styles.css">
```

Comment:
I'd suggest just inlining style.css instead with a `<style>` element... one less file to deal with.